packages/terra-layout/src/Layout.jsx

```diff
@@ -72,6 +72,17 @@ class Layout extends React.Component {
     window.removeEventListener('resize', this.updateSize);
   }
 
+  debounce(fn, delay) {
```

Comment:
Can we just use lodash's debounce package instead of rolling our own? https://www.npmjs.com/package/lodash.debounce